### PR TITLE
Clean up props

### PR DESCRIPTION
**Pull request by sastels**
delete a bunch of unused props, and hard coded questions.

### CODE REVIEW

File: __tests__/pages/favourites_page_test.js [hunk 1]
```diff
@@ -87,11 +86,11 @@ describe("Favourites Page", () => {
...
       selectedNeeds: {},
-      serviceType: "CAF",
+      // serviceType: "CAF",
       sortBy: "relevance",
-      patronType: "family",
-      statusAndVitals: "",
-      serviceHealthIssue: "",
+      // patronType: "family",
+      // statusAndVitals: "",
+      // serviceHealthIssue: "",
```
Comment (smcmurtry) [inline __tests__/pages/favourites_page_test.js]: 🔥

File: __tests__/pages/guided_test.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ describe("Guided", () => {
       },
       t: translate,
       storeHydrated: true,
-      dispatch: jest.fn(),
+      // dispatch: jest.fn(),
```
Comment (smcmurtry) [inline __tests__/pages/guided_test.js]: 🔥

File: pages/guided.js [hunk 3]
```diff
@@ -177,7 +175,7 @@ const mapStateToProps = reduxState => {
 Guided.propTypes = {
   reduxState: PropTypes.object,
   sectionOrder: PropTypes.array.isRequired,
-  dispatch: PropTypes.func,
+  // dispatch: PropTypes.func,
```
Comment (smcmurtry) [inline pages/guided.js]: 🔥

File: __tests__/pages/favourites_page_test.js (hunk 1 above)
Comment (sastels) [reply __tests__/pages/favourites_page_test.js]: 😞

**Review by smcmurtry — APPROVED** [review]
Nice!! 🦑

**Comment by sastels** [comment]
[User answers as little profile questions as possible](https://trello.com/c/MF0pMzGT/20-user-answers-as-little-profile-questions-as-possible)

**Pull request merged**